src/archivematicaCommon/lib/elasticSearchFunctions.py

```diff
@@ -206,20 +219,13 @@ def aip_mapping_is_correct(conn):
 
     return mapping['aipfile']['properties']['AIPUUID']['index'] == 'not_analyzed'
 
-# try up to three times to get a connection
-def connect_and_create_index(index, attempt=1):
-    if attempt <= 3:
-        conn = Elasticsearch(hosts=getElasticsearchServerHostAndPort())
-
-        response = conn.indices.create(index, ignore=400)
-        if 'error' in response and 'IndexAlreadyExistsException' in response['error']:
-            return conn
-
-        set_up_mapping(conn, index)
-        conn = connect_and_create_index(index, attempt + 1)
-    else:
-        conn = False
-
+def connect_and_create_index(index):
```

Comment:
As written, the original version could never have worked as expected. It appeared to rely on `put_mapping` silently failing, which may or may not have been possible with pyes but which is definitely not possible in the current configuration.